compiler/aarch64/codegen/ARM64Debug.cpp

```diff
@@ -1198,12 +1212,15 @@ TR_Debug::print(TR::FILE *pOutFile, TR::ARM64Trg1ImmInstruction *instr)
          }
       trfprintf(pOutFile, ", " POINTER_PRINTF_FORMAT, (instr->getBinaryEncoding() + offset));
       }
-   else if ((op == TR::InstOpCode::fmovimms) || (op == TR::InstOpCode::fmovimmd))
+   else if ((op == TR::InstOpCode::fmovimms) || (op == TR::InstOpCode::fmovimmd) ||
+            (op == TR::InstOpCode::vfmov4s) || (op == TR::InstOpCode::vfmov2d))
       {
       uint32_t imm = instr->getSourceImmediate() & 0xFF;
       trfprintf(pOutFile, ", 0x%02x (%lf)", imm, getDoubleFromImm8(imm));
       }
-   else
+   else if (op == TR::InstOpCode::movzx || op == TR::InstOpCode::movzw ||
+           op == TR::InstOpCode::movnx || op == TR::InstOpCode::movnw ||
+           op == TR::InstOpCode::movkx || op == TR::InstOpCode::movkw)
```

Comment:
The following two styles are mixed in the same function.
- `if (op == TR::InstOpCode::... || op == TR::InstOpCode::...)`
- `if ((op == TR::InstOpCode::...) || (op == TR::InstOpCode::...))`

Reply:
fixed it.

Reply:
Could you add one more space to L.1221-1222, please?

Reply:
Added a space to those lines.